### PR TITLE
Update ci-viewer members

### PR DESCRIPTION
**Pull request by yuzliu**
Add Dan Sun and Yuzhui Liu

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/internal-acls/181)

### CODE REVIEW

**Comment by yuzliu** [comment]
@googlebot I signed it!

File: ci-viewer.members.txt [hunk 1]
```diff
@@ -7,3 +7,5 @@ holden.karau@gmail.com
 iman.tabrizian@gmail.com
 nrchakradhar379@gmail.com
 redhat-team@kubeflow.org
+dsun20@bloomberg.net
```
Comment (jlewi) [inline ci-viewer.members.txt]: Alphabetical order please.

Comment (yuzliu) [reply ci-viewer.members.txt]: Updated:)

Comment (yuzliu) [reply ci-viewer.members.txt]: @jlewi let me know if this pr looks good to you :)

**Pull request merged**